src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewCell.cs

```diff
@@ -2597,6 +2613,42 @@ public virtual void InitializeEditingControl(int rowIndex, object initialFormatt
             }
         }
 
+        /// <summary>
+        /// The method checks that the string value of the cell is greater than its visible area
+        /// </summary>
+        /// <param name="rowIndex"> Index of row</param>
+        /// <param name="stringValue">String value of the cell</param>
+        private bool IsTextTruncated(int rowIndex, string stringValue)
```

Comment:
Since we're checking whether the text is fully visible or clipped by cell bounds, I don't think "truncate" is the right word, as it implies shortening of the text.
```suggestion
        private bool IsCellTextClipped(int rowIndex, string text)
```